python/utils.py

```diff
@@ -0,0 +1,230 @@
+"""
+A Python library to interface with FastFEC.
+
+This library provides methods to
+  * parse a .fec file line by line, yieling a parsed result
+  * parse a .fec file into parsed output .csv files
+"""
+
+import csv
+import datetime
+import logging
+import os
+import pathlib
+from ctypes import (CFUNCTYPE, POINTER, c_char, c_char_p, c_int, c_size_t,
+                    c_void_p, memmove)
+from dataclasses import dataclass
+from glob import glob
+
+logger = logging.getLogger("fastfec")
```

Comment:
Maybe I missed it, but I don't see a logging config around. Did you miss a commit?

Reply:
So, I need more direction on a logging config. Can you point me somewhere? (I thought the above _was_ the config)

Reply:
Ooops yes I owed you that. We have a few examples, that we copy and paste around and mildly tweak for each project. This is one: https://github.com/WPMedia/news-eng-census-api/blob/develop/census/logging.py.

There are a few nice things, like being able to set an environment variable to configure the logging level (we would want something more specific than `APP_LOG_LEVEL` IMO). We maybe do not need this now though? I feel like this PR has enough.

Reply:
Yea, I'm going to vote in favor of not doing this now since this PR is massive